tests/test_register.py

```diff
@@ -25,7 +25,7 @@ def register_settings(make_settings):
 
 
 def test_successful_register(register_settings):
-    """Test a successful package registration"""
+    """Registering a package succeeds for a valid repository url and package."""
```

Comment:
Sorry, the mood needs to be imperative. "Register a package successfuly ..." etc. We should slowly re-enable flake8-docstrings for these files as we rewrite the docstrings

Reply:
Okay, and then when it fails, should it be "Fail to register a package..." etc?

Reply:
Yes

Reply:
Cool, I have updated all docstrings to be a verb and have imperative mood.

Reply:
> We should slowly re-enable flake8-docstrings for these files as we rewrite the docstrings

Do we do this in a separate PR?

Reply:
> Do we do this in a separate PR?

Yes. I created #630 for discussion.